### PR TITLE
CNV-12348: RN - Added note about new virtctl vnc sub-command

### PR DESCRIPTION
**Pull request by sjhala-ccs**
This PR addresses [CNV-12348](https://issues.redhat.com/browse/CNV-12348)

Added note about the `--proxy-only` option for the `virtctl vnc` command

Preview: https://deploy-preview-33718--osdocs.netlify.app/openshift-enterprise/latest/virt/virt-4-8-release-notes.html#virt-4-8-new

### CODE REVIEW

**Comment by sjhala-ccs** [comment]
QE ack received in Jira

File: virt/virt-4-8-release-notes.adoc [hunk 1]
```diff
@@ -49,6 +49,9 @@ The SVVP Certification applies to:
 
 //CNV-12271 A new tech preview feature allows VM owners to hot-plug and hot-unplug disks with a running virtual machine
 
+//CNV-12348 Add `--proxy-only` option to `virtctl vnc` for only proxying a VNC connection to allow manual connections
```
Comment (ousleyp) [inline virt/virt-4-8-release-notes.adoc:52]: The only thing I'd suggest is to avoid anthropomorphic phrasing like "The `<non-human thing>` allows you..." though I realize we do this a lot in other docs. I might rephrase this as "You can use the `--proxy-only` option with the `virtctl vnc` command to..."

**Review by ousleyp — APPROVED** [review]
LGTM with one suggestion to take or leave.

File: virt/virt-4-8-release-notes.adoc (hunk 1 above)
Comment (sjhala-ccs) [reply virt/virt-4-8-release-notes.adoc:52]: Thanks for the suggestion. I am going to merge this as is.
